### PR TITLE
Get data api

### LINKED ISSUE #14: Get initial data for app
How will this application get initial data? At the moment, shows, venues, and artists must be added through the administrator console. Should users be able to add venues, artists, shows? If so, how will you moderate this? Where else will your application get data from? Maybe you could focus on the Twin Cities and find a list of venues; and figure out some programatic method of getting data for shows, perhaps write a web scraper for First-Avenue.com and other popular venues?
Should users be able to create shows from artists and venues already in the database?

### PR DESCRIPTION
**Pull request by YoussefMIbrahim**

### CODE REVIEW

**Comment by xd3262nd** [comment]
@YoussefMIbrahim I believe you might need to include the library that you used into the `requirements.txt`. I'm getting the error for the requests library

File: lmn/tests/test_show_Data.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
...idth":640,"height":360,"fallback":False},{"ratio":"3_2","url":"https://s1.ticketm.net/dam/a/300/88bcb3d0-aa78-428d-ad10-52514ea72300_570131_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":False},{"ratio":"3_2","url":"https://s1.ticketm.net/dam/a/300/88bcb3d0-aa78-428d-ad10-52514ea72300_570131_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":False}],"classifications":[{"primary":False,"segment":{"id":"KZFzniwnSyZfZ7v7na","name":"Arts & Theatre"},"genre":{"id":"KnvZfZ7v7l1","name":"Theatre"},"subGenre":{"id":"KZazBEonSMnZfZ7vAve","name":"Musical"},"type":{"id":"KZAyXgnZfZ7v7nI","name":"Undefined"},"subType":{"id":"KZFzBErXgnZfZ7v7lJ","name":"Undefined"},"family":False}],"upcomingEvents":{"_total":495,"tmr":283,"ticketmaster":212},"_links":{"self":{"href":"/discovery/v2/attractions/K8vZ9174wRf?locale=en-us"}}}]}}]},"_links":{"first":{"href":"/discovery/v2/events?city=Minneapolis&page=0&size=1"},"self":{"href":"/discovery/v2/events?size=1&city=Minneapolis"},"next":{"href":"/discovery/v2/events?city=Minneapolis&page=1&size=1"},"last":{"href":"/discovery/v2/events?city=Minneapolis&page=198&size=1"}},"page":{"size":1,"totalElements":199,"totalPages":199,"number":0}}
```
Comment (xd3262nd) [inline lmn/tests/test_show_Data.py:15]: Probably save this in a json file or something like fixtures so can keep the test file clean

**Comment by BrettKoenig04** [comment]
I'm getting a requests error when attempting to run a local server.

**Review by xd3262nd — APPROVED** [review]
@YoussefMIbrahim  Not sure if you are still trying to work more on this, but as far as I know the code is running alright. So I will go ahead and approve this unless you are planning to do some cleanup works.

**Pull request merged**